pptx2md/parser.py

```diff
@@ -166,5 +166,6 @@ def parse(prs, outputer):
                 process_text_block(shape)
             elif shape.shape_type == MSO_SHAPE_TYPE.PICTURE:
                 process_picture(shape)
-
+    out.flush()
```

Comment:
maybe we do not need to do flush manually as calling close implies flushing: https://docs.python.org/3/library/io.html#io.IOBase.close